queries/lalrpop/highlights.scm

```diff
@@ -72,7 +72,14 @@
 ] @punctuation.delimiter
 
 (lifetime
-  (identifier) @keyword.storage)
+  "'" @keyword.modifier)
+
+(lifetime
+  (identifier) @type.modifier)
+
+(lifetime
+  (identifier) @keyword.modifier
+  (#any-of? @keyword.modifier "static" "_"))
```

Comment:
Make them all the same.

Reply:
`@type.modifier` or `@keyword.modifier`?